.gitignore

```diff
@@ -4,7 +4,12 @@
 # production
 /dist
 
-# misc
-.vscode
-
 npm-debug.log*
+
+# hidden files
+.*
+
+# whitelist hidden files
+!.gitignore
+!.prettierrc
+!.husky
```

Comment:
husky
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣀⣠⣤⣤⣤⣤⣀⣀⣀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣤⡶⠟⠛⠋⠉⠉⠉⠁⠀⠀⠀⠉⠉⠙⠛⠻⠶⢦⣤⣀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣴⠞⠋⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠙⠛⢷⣤⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣤⠾⠋⠀⢠⣄⣀⣤⣤⣤⣤⣤⣄⠀⠀⠀⠀⠀⠀⠀⢠⣤⣤⣤⣤⣤⣀⣀⡀⠀⠙⠻⣶⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣴⠟⠁⠀⠀⠀⣿⣿⣿⣿⣿⠿⠿⠿⠛⠀⠀⠀⠀⠀⠀⠀⠈⠛⠿⠿⣿⣿⣿⣿⣿⠀⠀⠀⠈⠻⣦⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⠟⠁⠀⠀⠀⠀⠀⠈⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠟⠿⠀⠀⠀⠀⠀⠈⠻⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣼⠏⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⠀⠀⠀⠀⠀⠀⠀⠀⠙⣷⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⢀⣾⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠳⣄⠀⠀⠀⠀⠀⠀⠀⠘⣷⡀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⣼⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣀⣀⣠⣤⣄⣀⣀⣀⡀⠀⠀⠀⠀⠀⣀⣀⣀⣀⣀⣠⣤⣀⣀⣀⠙⣧⠀⠀⠀⠀⠀⠀⠰⡜⣷⡄⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⢸⡏⠀⠀⠀⠀⠀⠀⠀⢀⣴⡿⠛⠉⠉⠈⣭⣿⣿⣿⣯⡙⠛⢶⣦⡶⠛⠛⣿⣿⣿⣿⣏⠀⠀⠉⠉⠻⣿⡆⠀⠀⠀⠀⠀⠀⣿⣿⣷⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⣿⠁⠀⠀⠀⠀⠀⠀⠀⣼⣿⠀⠀⠀⠀⠀⣿⣿⣿⣿⣿⠇⠀⢀⣿⠀⠀⠸⣿⣿⣿⣿⣿⠃⠀⠀⠀⢀⣾⣷⡆⠀⠀⠀⠀⠀⢸⣿⣿⡇⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢰⣿⠀⠀⠀⠀⠀⠀⠀⠀⢷⡛⣧⡀⠀⠀⠀⠘⠻⡿⠿⠋⠀⠀⣼⠿⣧⡀⠀⠙⠻⠿⠿⠋⠀⠀⠀⢀⣾⡟⣿⠁⠀⠀⠀⠀⠀⠀⣿⣿⣇⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⡟⠀⠀⠀⠀⠀⠀⠀⠀⠈⢷⣌⠿⣦⣀⠀⠀⠀⠀⣀⣀⣤⡾⠋⠀⠙⢿⣦⣀⠀⠀⠀⠀⠀⣀⣴⡿⠋⡼⠃⠀⠀⠀⠀⠀⠀⠀⣿⣿⣿⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠹⠷⣌⠛⠿⠶⠶⠿⠿⠛⠉⠀⠀⠀⠀⠀⠉⠛⠻⠿⠷⠾⠿⠟⠋⠠⠞⠁⠀⠀⠀⠀⠀⠀⠀⢀⣿⣯⣿⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⣿⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠉⠑⠒⠂⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠒⠒⠂⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸⣿⣿⣿⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢸⣿⣿⡄⠀⠀⠀⠀⣀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⠀⠀⠀⠀⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀
⠀⠀⢀⣠⣶⡶⣾⣿⣿⣷⡄⠀⠀⠀⠻⠿⠲⠶⢦⣤⣤⣀⣀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣀⣤⣤⡴⠶⠖⢿⠟⠀⠀⠀⣤⣿⣿⣿⣥⣶⣶⣄⠀⠀⠀
⠀⠀⣸⣿⡿⣶⣿⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠛⠛⠛⠒⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠛⠛⠛⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸⣿⣿⣿⣿⠾⢿⣿⡇⠀⠀
⠀⠀⣿⠏⠀⠀⠀⠈⢿⣿⣷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠶⢶⣤⣤⣄⣀⣀⡀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⣿⣿⡿⠋⠀⠀⠀⠹⣷⠀⠀
⠀⣼⡏⠀⠀⠀⠀⠀⠈⢿⣿⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣴⣿⣿⡿⠁⠀⠀⠀⠀⠀⢹⡇⠀
⠀⣿⠁⠀⠀⠀⠀⠀⠀⠘⣿⣿⣆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣰⣿⣿⣿⠁⠀⠀⠀⠀⠀⠀⠈⣿⠀
⠀⣿⠀⢀⣰⣶⣄⠀⠀⠀⣿⡎⠻⣷⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣿⣿⠟⢡⣿⠀⠀⠀⢀⣤⣄⠀⠀⣿⠀
⢸⡏⠀⠀⠉⣽⣿⡀⠀⠀⢸⣧⠀⠈⠻⣷⣦⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣴⣾⣿⡿⠋⠀⢸⡇⠀⠀⠀⢸⣏⠀⠀⠀⢻⡆
⢸⡇⠀⠀⠛⠻⣿⣇⠀⠀⢀⣿⠀⠀⠀⠈⠻⢿⣷⣦⣄⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣠⣴⣾⣿⡿⠟⠁⠀⠀⠀⣸⠃⠀⠀⢀⣿⣿⠃⠀⠀⢸⡇
⢸⡇⠀⠀⠶⣶⡾⠿⠶⣶⠾⠟⠀⠀⠀⠀⠀⠀⠈⠙⠻⢿⣿⣷⣶⣶⣤⣤⣀⣀⣀⣀⣀⣀⣤⣤⣴⣶⣾⣿⣿⣿⠿⠛⠉⠀⠀⠀⠀⠀⠀⠸⠷⢶⣶⡿⣋⣿⠂⠀⠀⣸⠇
⠀⣿⡄⠀⠀⣸⡷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠙⠛⠛⠛⠛⠛⠛⠙⠛⠛⠛⠛⠋⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⢻⣿⡀⠀⢀⣿⠁
⠀⠘⠿⠿⠟⠛⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠙⠿⠶⠿⠋⠀

Reply:
I know it's not husky's fault it's just funny to see you adding it.